src/go/configinfo/service_info.go

```diff
@@ -719,6 +719,13 @@ func (s *ServiceInfo) processLocalBackendOperations() error {
 }
 
 func (s *ServiceInfo) processUsageRule() error {
+	// hard code to allow health check, if defined, to bypass service control,
+	for _, methodName := range util.HardCodedSkipServiceControlMethods {
```

Comment:
please add comment: If user desires to enable service control for one of these allowlisted methods, they can specify `skip_service_control: false` in their service config and ESPv2 honors it.